lib/between_meals/repo/git.rb

```diff
@@ -138,7 +138,7 @@ def _config key
         cmd = Mixlib::ShellOut.new("#{@bin} config #{key}")
         cmd.run_command
         cmd.stdout.strip
-      rescue
+      rescue => e
```

Comment:
I'm gonna clean this file up in separate commit, along with refactoring and rubocop